climpred/tests/test_smoothing.py

```diff
@@ -133,3 +99,108 @@ def test_compute_after_smooth_goddard_2013(
     actual = compute_perfect_model(PM_da_initialized_3d_full, PM_da_control_3d_full)
     north_atlantic = actual.sel(lat=slice(40, 50), lon=slice(-30, -20))
     assert not north_atlantic.isnull().any()
+
+
+@pytest.mark.parametrize('tsmooth_kws', [{'lead': 2}, {'lead': 4}, {'time': 4}])
+def test_HindcastEnsemble_temproal_smooth_leadrange(hindcast_recon_3d, tsmooth_kws):
+    he = hindcast_recon_3d
+    dim = list(tsmooth_kws.keys())[0]
+    he = he.smooth(smooth_kws=tsmooth_kws)
+    assert he._datasets['initialized'].lead.attrs['units']
+    skill = he.verify(metric='acc')
+    assert skill.lead[0] == f'1-{1+tsmooth_kws[dim]-1}'
+
+
+@pytest.mark.parametrize('smooth', [2, 4])
+@pytest.mark.parametrize(
+    'pm',
+    [
+        pytest.lazy_fixture('perfectModelEnsemble_initialized_control_1d_ym_cftime'),
+        pytest.lazy_fixture('perfectModelEnsemble_initialized_control_1d_mm_cftime'),
+        pytest.lazy_fixture('perfectModelEnsemble_initialized_control_1d_dm_cftime'),
+    ],
+)
+def test_PerfectModelEnsemble_temporal_smoothing_cftime_and_skill(pm, smooth):
+    """Test that PredictionEnsemble.smooth({'lead': int}) aggregates lead."""
+    pm = pm.isel(lead=range(6))
+    he_smoothed = pm.smooth({'lead': smooth})
+    assert (
+        he_smoothed.get_initialized().lead.size
+        == pm.get_initialized().lead.size - smooth + 1
+    )
+    assert isinstance(he_smoothed._temporally_smoothed, dict)

```

Comment:
assert `pm_smoothed._temporally_smoothed` if you implement the suggestion I had on having a default value